README.md

```diff
@@ -103,6 +103,17 @@ export default App;
 
 You can find documentation and examples of all our components in this [storybook](https://components.gnosis-safe.io/).
 
+## Testing
+
+Snapshot tests are generated automatically from the Storybook stories.
+
+To run the tests locally:
+```
+yarn test --watch
```

Comment:
can we add a little more detailed explanation on how it works?
for example:
- Run the project as always and do all the modifications that you need.
- Run `yarn test` or `yarn test --watch` to re-run the test for each modification you do
- Once you finished working you need to update the `storyshoots` to be aware of the new changes, to do so, run `yarn test -u`. When the command finishes, the outcome will be the source of truth.